prime-angular-frontend/src/app/shared/components/enrollee/enrollee-review/enrollee-review.component.html

```diff
@@ -116,6 +116,8 @@
   <ng-container *ngFor="let careSetting of enrolment.careSettings">
 
     <app-enrollee-property title="Care Setting"
+                           [hasError]="enrolmentErrors?.missingHealthAuthorityCareSetting && careSetting.careSettingCode === CareSettingEnum.HEALTH_AUTHORITY"
+                           errorMessage="Must select health authorities when selecting health authority as your care setting."
```

Comment:
Just a suggestion, if we can sentence-case the "Health Authority" part since it's how it shows in the dropdown? as in 

`Must select health authorities when selecting Health Authority as your care setting.`

Also using **select** twice makes reading the error a little awkward, I'm just throwing this in for example to clarify for the users:

`Must specify the health authorities you work at when selecting Health Authority as your care setting.`

Reply:
updated the test